app-citizen/src/app/shared/gray-option/gray-option.component.html

```diff
@@ -1,4 +1,4 @@
-<div (click)="onClick()" class="gray-option">
+<div class="gray-option">
```

Comment:
Pero entonces como se ejecuta el método que iba asociado onClick()?

Reply:
Revisando de nuevo el problema, he visto que se estaba usando un alias en el @Output con el mismo nombre ('click') del evento que lanza y eso parece ser que produce ese comportamiento.

@Outputs should not be renamed (no-output-rename)tslint(1)